code/game/objects/items.dm

```diff
@@ -116,6 +124,16 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 	if(GLOB.rpg_loot_items)
 		rpg_loot = new(src)
 
+	LAZYINITLIST(default_strings)
+	//populate default_strings
+	default_strings += "very low"
+	default_strings += "low"
+	default_strings += "medium"
+	default_strings += "high"
+	default_strings += "robust"
+	default_strings += "very robust"
+	default_strings += "exceptionally robust"
+

```

Comment:
`There are three issues with it. One is an implementation problem, you're storing a list of strings on every single obj/item in the game, which is a non-trivial amount of lists to have floating around.

One of the bigger problems is that by storing the strings forever, they're sitting around in the string tree forever.

And finally, there's literally no gain from storing the strings like this. What's the idea, admins var editing the list? I don't get it.`I vastly preferred it when you were just had the strings inlined into the function like it was before.